src/test/java/be/seeseemelk/mockbukkit/block/state/BlockStateMockTest.java

```diff
@@ -18,6 +13,8 @@
 import be.seeseemelk.mockbukkit.WorldMock;
 import be.seeseemelk.mockbukkit.block.BlockMock;
 
+import static org.junit.Assert.*;
```

Comment:
Ugh, wildcard imports are generally not the best idea.

Reply:
Sorry, IntelliJ autochanged it, I should turn that off.